pytest_inmanta/parameters.py

```diff
@@ -96,6 +96,14 @@
     legacy=inm_mod_repo_legacy,
 )
 
+pip_index_url = ListTestParameter(
+    argument="--pip-index-url",
+    environment_variable="INMANTA_PIP_INDEX_URL",
+    usage=("List of project-wide pip indexes"),
```

Comment:
```suggestion
    usage="List of project-wide pip indexes",
```
These are redundant.